README.md

```diff
@@ -28,6 +28,8 @@ This command generates static content into the `build` directory and can be serv
 
 ## Deployment
 
+You **should** be using SSH, but if you are not, you need to define:
```

Comment:
As per the documentation: <img width="1497" alt="image" src="https://user-images.githubusercontent.com/8750702/152220368-f68f15c1-19e7-4346-b09a-5bf85ee87f64.png">